app/javascript/chat/chat.jsx

```diff
@@ -1283,6 +1291,7 @@ export default class Chat extends Component {
               channelsLoaded={state.channelsLoaded}
               filterQuery={state.filterQuery}
               expanded={state.expanded}
+              aria-expanded={state.expanded}
```

Comment:
I added this as it adds or removes this DOM element from the accessibility tree.